proto/ibc/core/channel/v1/channel.proto

```diff
@@ -90,6 +90,21 @@ enum Order {
   ORDER_ORDERED = 2 [(gogoproto.enumvalue_customname) = "ORDERED"];
 }
 
+// IBCVersion defines the version that the IBC packet intends to use.
+// This allows asynchronous upgrades over time as unsupported IBC Versions
+// will simply be rejected by the receiver and timed out on sender.
+enum IBCVersion {
+  option (gogoproto.goproto_enum_prefix) = false;
+
+  // zero-value for IBC version.
+  // This will be treated as a classic packet
+  IBC_VERSION_UNSPECIFIED = 0;
```

Comment:
Hmm, should we assign classic to 0? Would be encoding compliant with the existing structure, I'm fairly sure (empty fields not encoded), but it doesn't matter so much since this is just go api

Reply:
Proto makes me suffix this enum with UNSPECIFIED. idk if it can be overridden.

Agreed, this is just inside the chain doesn't go over the protocol so it doesn't matter

Reply:
https://github.com/cosmos/ibc-go/blob/4a0da8f37d309ea52e089b67c6eda929ccad35ae/proto/ibc/core/channel/v1/channel.proto#L85

Reply:
https://github.com/cosmos/ibc-go/pull/6986#discussion_r1699154395